src/colp/components/manual_correction_report.py

```diff
@@ -16,11 +16,16 @@ def __call__(self):
         ):
             st.info("No Manual Correction Table Available.")
             return
-        self.display_manual_correction()
+        elif self.modifications_applied is None:
+            self.display_not_applied_correction()
+        elif self.modifications_not_applied is None:
+            self.display_applied_correction()
+        else:
+            self.display_applied_correction()
+            self.display_not_applied_correction()
```

Comment:
I thought it might be easier to have the check for the table being none or empty the associated methods? So for example `display_manual_correction` checks if the df is none. If it's none, tell the user and return. Then check if it's empty, same thing. Then finally display the content is there's data to display. 
This process should happen separately for the two sections I think

Reply:
Great suggestion! I have made the change.